src/unity/python/turicreate/_sys_util.py

```diff
@@ -64,7 +63,8 @@ def make_unity_server_env():
 
     # Set mxnet envvars
     if 'MXNET_CPU_WORKER_NTHREADS' not in env:
-        num_workers = min(2, int(env.get('OMP_NUM_THREADS', _sys_info.NUM_CPUS)))
+        from multiprocessing import cpu_count
```

Comment:
Was the `minipsutil` thing plumbed through to `sys_info` before (is that where `NUM_CPUS` was coming from)? If so, is there any chance that `multiprocessing.cpu_count` would give a different number of CPUs? Or was the now-defunct server-client architecture the only reason we had the CPU counting logic on the C++ side before?

Reply:
Yes, `minisputil` was plumbed through to `sys_info`.

If I recall correctly, `minisputil` is remnant of jobs and task. 

I think this should be safe; I would trust something in the python standard library to give the right answer, more than I would trust the code in `minisputil`. It's only anecdotal evidence but they give the same answer on machine.